### DISCUSSION TITLE
Winfrom中浏览页面缩放问题

### DISCUSSION THREAD

**Discussion by 1713237685** [cefsharp/CefSharp#4141] [General]
怎样实现加载出来的可以进行放大缩小

**Comment by amaitland** [comment]
English please.

**Reply by 1713237685** [reply]
How to realize the mouse wheel zoom in and out of the page in WinForm

**Reply by amaitland** [reply]
There's a discussion about this at https://magpcss.org/ceforum/viewtopic.php?f=6&t=16480


- https://github.com/cefsharp/CefSharp/blob/cefsharp/102/CefSharp.WinForms.Example/BrowserTabUserControl.cs#L338
- https://github.com/cefsharp/CefSharp/wiki/Frequently-asked-questions#JSEvent

Hooking the event in JavaScript is probably easier, might look something like the following  

```js  
        // Ctrl+Mousewheel handler for zoom in and out...
window.onwheel=function(event)
{
          if (event.ctrlKey){
           if (event.wheelDelta < 0){
        
              // Use CefSharp.PostMessage to notify zoom out;
            }
            else if (event.wheelDelta > 0){
        
              // Use CefSharp.PostMessage to notify zoom in
            }
          }
 };
```
